app/Enums/HomePageTabs.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Enums;
+
+enum HomePageTabs: string
+{
+    case Feed = 'feed';
```

Comment:
Recent
Following

Reply:
If we change "feed" to "recent", we have to change the route name, URL and probably file names too.
You want me to proceed with that ?